alf/data_structures.py

```diff
@@ -99,7 +99,9 @@ class TimeStep(
     - reward: a ``Tensor`` of reward values from executing 'prev_action'.
     - discount: A discount value in the range :math:`[0, 1]`.
     - observation: A (nested) ``Tensor`` for observation.
-    - prev_action: A (nested) ``Tensor`` for action from previous time step.
+    - prev_action: A (nested) ``Tensor`` for action from previous time step. Note that
```

Comment:
"for action generated for the previous time step"

Reply:
Updated comments